certified-connectors/Connect2All On-Premises/apiDefinition.swagger.json

```diff
@@ -0,0 +1,4024 @@
+{
+  "basePath": "/",
+  "consumes": [],
+  "definitions": {
+    "CodeunitId": {
+      "description": "The codeunit you wish to execute",
+      "format": "int32",
+      "title": "",
```

Comment:
@Joerio empty properties are not allowed. Please look through empty titles and add value to them.

Reply:
Added titles to all empty title properties.